src/common/components/episode-list/episode-list.vue

```diff
@@ -71,7 +71,7 @@ export default class EpisodeList extends Vue {
 
   public $refs!: {
     container: Vue
-    episodes: Episode[]
+    episodes: Episode
```

Comment:
Is this type really correct? Don't you get an array of refs when you `ref` a `v-for` element?

Reply:
I kept getting undefined on the array, it seemed as though the ref only applied to one element and trying to use it as an array is what was breaking it.

Reply:
Ah, it seems that the virtual scroll solution does not set every item's ref, only one of them. If possible, it's probably best to remove that ref altogether

Reply:
In the comment above I mentioned the scrollToItem function, that would eliminate the need for this ref as well.